manager/integration/tests/test_node.py

```diff
@@ -498,12 +498,6 @@ def test_replica_scheduler_too_large_volume_fit_any_disks(client):  # NOQA
     6. Attach the volume.
     7. Make sure every replica landed on different nodes's default disk.
     """
-
-    over_provisioning_setting = client.by_id_setting(
-        SETTING_STORAGE_OVER_PROVISIONING_PERCENTAGE)
-    client.update(over_provisioning_setting,
-                  value=DEFAULT_STORAGE_OVER_PROVISIONING_PERCENTAGE)
-
```

Comment:
If we remove this setting, I think we need to add some logic to pytest.fixture client to reset the over_provisioning_setting to DEFAULT_STORAGE_OVER_PROVISIONING_PERCENTAGE before each test. Otherwise, this setting might have value from previous tests, which makes the tests like this one flaky. WDYT?

Reply:
Can you elaborate on how this setting would have value from previous tests?

IIRC, the client fixture already includes the [cleanup_client](https://github.com/longhorn/longhorn-tests/blob/83d3cf9178696830587792cd081541fc89cb5ad9/manager/integration/tests/common.py#L1548-L1550) which [resets the settings](https://github.com/longhorn/longhorn-tests/blob/83d3cf9178696830587792cd081541fc89cb5ad9/manager/integration/tests/common.py#L1599) to their [default values](https://github.com/longhorn/longhorn-tests/blob/83d3cf9178696830587792cd081541fc89cb5ad9/manager/integration/tests/common.py#L3240). Currently, the default value is set to [100](https://github.com/longhorn/longhorn-manager/blob/4410d5c1a807928de4c9049a9578a074d6a70813/types/setting.go#L458) which is the same as the DEFAULT_STORAGE_OVER_PROVISIONING_PERCENTAGE.